<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1594</width>
    <height>898</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(217, 75, 154, 255), stop:1 rgba(219, 255, 174, 255))}</string>
    </property>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>40</y>
       <width>201</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 36pt &quot;예스 고딕 레귤러&quot;;</string>
     </property>
     <property name="text">
      <string>분석 결과</string>
     </property>
    </widget>
    <widget class="QLabel" name="video_result">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>400</y>
       <width>511</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;예스 고딕 레귤러&quot;;
color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>비디오 분석 내용</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QPushButton" name="to_welcome">
     <property name="geometry">
      <rect>
       <x>1380</x>
       <y>750</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:60px;
background-color:rgb(217, 140, 121);
font: 16pt &quot;예스 고딕 레귤러&quot;;
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>시작화면으로</string>
     </property>
    </widget>
    <widget class="QPushButton" name="to_rewatch">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>740</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:60px;
background-color:rgb(217, 140, 121);
font: 16pt &quot;예스 고딕 레귤러&quot;;
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>다시보기</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_video">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>180</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 36pt &quot;예스 고딕 레귤러&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;목소리&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_audio">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>180</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 36pt &quot;예스 고딕 레귤러&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;목소리&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="video_suggestion">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>280</y>
       <width>341</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;예스 고딕 레귤러&quot;;
color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>좀 더 웃으시면 좋겠네요!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="audio_suggestion">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>280</y>
       <width>341</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;예스 고딕 레귤러&quot;;
color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>좀 더 침착하게!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="audio_result">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>400</y>
       <width>511</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;예스 고딕 레귤러&quot;;
color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>오디오 분석 내용</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
